codegen/smithy-dafny-codegen-modules/smithy-python/codegen/smithy-python-codegen/src/main/java/software/amazon/smithy/python/codegen/DirectedPythonCodegen.java

```diff
@@ -291,7 +291,11 @@ private void generateInits(CustomizeDirective<GenerationContext, PythonSettings>
             var initPath = entry.getKey().resolve("__init__.py");
             if (!entry.getValue().contains(initPath)) {
                 directive.fileManifest().writeFile(initPath,
-                        "# Code generated by smithy-python-codegen DO NOT EDIT.\n");
+                    """
+                    # Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+                    # SPDX-License-Identifier: Apache-2.0
+                    # Do not modify this file. This file is machine generated, and any changes to it will be overwritten.\n
+                    """);
```

Comment:
I wonder if it would be reasonable to add a hook to `PythonIntegration` for customizing this...

Reply:
I definitely think that is the "right" way to do this...